js/onoffice-captchacontrol.js

```diff
@@ -20,4 +20,21 @@ var onOffice = onOffice || {};
 
 		return iePosition > 0 || !!navigator.userAgent.match(/Trident.*rv\:11\./);
 	};
-})();
\ No newline at end of file
+})();
+
+var requiresCaptchaForm = requiresCaptchaForm || false;
```

Comment:
Maybe this boolean is not needed, see comments in Form.php and CaptchaDataChecker.php.